exercises/03_reverse/reverse.go

```diff
@@ -1,4 +1,19 @@
 package main
 
+import "fmt"
+
+func reverse(n int) int {
+	y := 0
```

Comment:
Don't use short assignment `:=` for declaring variables with default values assigned. Use `var y int` declaration instead.